frontend/src/exploration/previews/layoutsData.ts

```diff
@@ -0,0 +1,42 @@
+import { ExplorationWeight } from '../../stores/exploration';
+import LayoutDefinition from './LayoutDefinition';
+
+const layoutsData: Record<keyof ExplorationWeight, LayoutDefinition> = {
+  C: {
+    filename: 'structural-layout.png',
+    description: 'Structural Layout',
+    path: '/visualization/graph',
+  },
+  BC: {
+    filename: 'empty-layout.png',
+    description: 'Betweenness Centrality',
+    path: '/visualization/schema', // TODO: change when layout added
```

Comment:
```suggestion
    path: '/visualization/schema', // TODO #299: Add missing visualisation preview links
```
(for every TODO in this file)